modules/rosa-sts-terraform-prerequisites.adoc

```diff
@@ -0,0 +1,102 @@
+ifeval::["{context}" == "rosa-understanding-terraform"]
+:tf-full:
+endif::[]
+
+:_content-type: CONCEPT
+[id="rosa-sts-terraform-prerequisites_{context}"]
+ifdef::tf-full[]
+= Prerequisites for Terraform
+endif::tf-full[]
+ifndef::tf-full[]
+.Prerequisites
+endif::tf-full[]  
+
+To use link:https://registry.terraform.io/providers/terraform-redhat/rhcs/latest/docs[the Red Hat Cloud Services provider] inside your Terraform configuration, you must meet the following prerequisites:
+
+* You have your offline link:https://console.redhat.com/openshift/token/rosa[OCM token].
```

Comment:
I suggest also to recommend the user to install ROSA CLI, with this instructions: https://docs.openshift.com/rosa/cli_reference/rosa_cli/rosa-get-started-cli.html